File: data/museum/oseg_generated/python/GetTicketCode_default_example.py
from pprint import pprint

from openapimuseum_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    try:
        api_caller = api.TicketsApi(api_client)

        response = api_caller.get_ticket_code(
            ticket_id="a54a57ca-36f8-421b-a6b4-2e8f26858a4c",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Tickets#get_ticket_code: %s\n" % e)

File: data/museum/oseg_generated/csharp/GetTicketCode_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPIMuseum.Api;
using Org.OpenAPIMuseum.Client;
using Org.OpenAPIMuseum.Model;

public class GetTicketCodeDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            var apiCaller = new TicketsApi(config);

            var response = apiCaller.GetTicketCode(
                ticketId: "a54a57ca-36f8-421b-a6b4-2e8f26858a4c"
            );
            var fileStream = File.Create("file_response.zip");
            response.Seek(0, SeekOrigin.Begin);
            response.CopyTo(fileStream);
            fileStream.Close();
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Tickets#GetTicketCode: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
